docs/guides/basic-compositions/build-composition.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Build your Composition
+tocHidden: true 
+weight: 260
+indent: true
+---
+
+# Build your Composition
+
+This is the third part of a four part tutorial to build an API for provisioning pre-configured VMs on top of the GCP Compute Engine service. If you have not already, you need to first [Build your XRD](build-xrd.md) first. In this part of the tutorial, you will build the Composition portion of your API.

```

Comment:
Use `[Build your XRD]({{<ref "master/build-xrd.md" >}})` links styles to detect broken links in the future. 

More details here: https://docs.crossplane.io/master/contributing/docs/#links